UI.py

```diff
@@ -237,18 +237,32 @@ def CreateGetLeavesAcrossRangeOptions():
 
     return member_options
 
-def CreateMultipleLeavesEmbed(leaves_group, include_reason):
-    embed = discord.Embed(
-        title = f'Applied Leaves',
-        description = f'<@!{leaves_group[0][0]["member_id"]}> has requested:',
-        colour = 0x4682B4
-    )
-    embed.set_thumbnail(url = os.getenv("Leave_Balance_Link"))
-    embed.add_field(name = '\u200B', value = '\u200B', inline = False)
+def CreateMultipleLeavesEmbed(leaves, include_reason):
+    embeds_to_send = []
+    for leaves_group in leaves:
+        embed = discord.Embed(
+            title = f'Applied Leaves',
+            description = f'<@!{leaves_group[0][0]["member_id"]}> has requested:',
+            colour = 0x4682B4
+        )
+        embed.set_thumbnail(url = os.getenv("Leave_Balance_Link"))
+        embed.add_field(name = '\u200B', value = '\u200B', inline = False)
+        embed = DesignMultipleLeavesEmbed(leaves_group, embed, include_reason)
+        if (embed == None):
+            continue
+
+        footer_text = (("\u200B " * 150) + datetime.date.today().strftime("%d/%m/%Y")) # magic number 150
```

Comment:
Don't do magic numbers. It's that simple. You even acknowledged that this is a magic number. How did this get submitted?

Reply:
this number is for design purpose, so it will be a magic number no matter how we look at it, but in order to clarify what this number is we can make a var and use it across the script.

PS: this conversation was done before on the discord voice channel, and there was no issue back then using this magic number

Reply:
I don't think you understand. Magic numbers refer to inline numbers directly used in algorithms. Declaring a variable with a suitable name makes it no longer a magic number.